crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -113,33 +114,29 @@ fn fragment(
         pbr_input.material.metallic = metallic;
         pbr_input.material.perceptual_roughness = perceptual_roughness;
 
+        // occlusion
         // TODO: Split into diffuse/specular occlusion?
         var occlusion: vec3<f32> = vec3(1.0);
 #ifdef VERTEX_UVS
         if ((pbr_bindings::material.flags & pbr_types::STANDARD_MATERIAL_FLAGS_OCCLUSION_TEXTURE_BIT) != 0u) {
             occlusion = vec3(textureSampleBias(pbr_bindings::occlusion_texture, pbr_bindings::occlusion_sampler, uv, view.mip_bias).r);
         }
 #endif
-#ifndef DEFERRED_PREPASS
 #ifdef SCREEN_SPACE_AMBIENT_OCCLUSION
         let ssao = textureLoad(screen_space_ambient_occlusion_texture, vec2<i32>(in.position.xy), 0i).r;
         let ssao_multibounce = gtao_multibounce(ssao, pbr_input.material.base_color.rgb);
         occlusion = min(occlusion, ssao_multibounce);
-#endif // SCREEN_SPACE_AMBIENT_OCCLUSION
-#endif // DEFERRED_PREPASS
+#endif
```

Comment:
Keep the screen space ao comment?

Reply:
On the endif? I can if you want, it’s only 3 lines from the ifdef, not nested, and would be the only such comment in the file though…

Reply:
Ok. No worries. It was a thought because it was removed. I kind of like having them there for clarity. But it doesn’t matter.